### DISCUSSION TITLE
设计模式学习之单件模式 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#241] [Announcements]
https://mazhuang.org/2014/05/31/HFDP-Note4/

**Comment by Jhaihao** [comment]
请问一下,这个评论这里怎么初始化  或者是 隐藏. .

**Comment by mzlogin** [comment]
> @Jhaihao
> 请问一下,这个评论这里怎么初始化  或者是 隐藏. .

源码 _config.yml 文件，第 104 - 109 行，改成你自己的配置，你的博客目前用的还是我的配置。
